src/main/target/ANYFCF7/target.h

```diff
@@ -24,6 +24,7 @@
 #define USBD_PRODUCT_STRING "AnyFCF7"
 
 #define USE_DSHOT
+#define USE_ESC_TELEMETRY
```

Comment:
@martinbudden Looks like I was not clear enough, I meant doing the above only once, and in `common.h`. I would argue that not having to explicitly add `#define USE_ESC_TELEMETRY` to every target makes it less prone to error when adding new targets, but there is pluses and minuses to both ways of doing it.